### DISCUSSION TITLE
Rabbitmq 3.8.35 and erlang 25

### DISCUSSION THREAD

**Discussion by Paxa** [rabbitmq/rabbitmq-server#6409] [Other] [answered]
I upgrade cluster from 3.8.9 to 3.9.35 following instruction https://www.rabbitmq.com/install-debian.html#apt-cloudsmith

And it automatically pick erlang 25, is it expected?

```
rabbitmq-diagnostics cluster_status
...
rabbit@vm1-01.internal: RabbitMQ 3.8.35 on Erlang 25.1.2
rabbit@vm1-02.internal: RabbitMQ 3.8.9 on Erlang 23.1.1
rabbit@vm1-02.internal: RabbitMQ 3.8.9 on Erlang 23.1.1
```

**Comment by Paxa** [comment]
In here it says that 3.8.x compatible with erlang from 23.2 to 24.3 
https://www.rabbitmq.com/which-erlang.html#eol-series

**Comment by michaelklishin** [comment, marked as answer]
RabbitMQ no longer caps the maximum supported Erlang version, [including for latest 3.8](https://github.com/rabbitmq/rabbitmq-packaging/blob/v3.8.x/debs/Debian/debian/control#L9) (which [has reached end of life](https://www.rabbitmq.com/versions.html))

Those who want to always use the exact version can use [version pinning](https://rabbitmq.com/install-debian.html#apt-pinning) which will break when the pinned version is phased out of support (like Erlang 23 had been earlier this year).

By far the best option is stay on the latest minor release and upgrade it to the latest available patch release reasonably recently. That automatically means support for the two latest major Erlang release series.

**Reply by Paxa** [reply]
Thanks
